### ISSUE TITLE
I think there is a problem with rollup-plugin-uglify?

### ISSUE THREAD

**Issue by lowsprofile** [developit/microbundle#35]
cc: @developit I think there is a problem with `rollup-plugin-uglify`

When I try to build my module this was comes up.
```sh
$ yarn microbundle
yarn run v1.1.0
$ microbundle build
Using named and default exports together. Consumers of your bundle will have to use jaysn['default'] to access
the default export, which may not be what you want. Use `exports: 'named'` to disable this warning
(node:4720) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Error transforming bundle with 'uglify' plugin: Unexpected token: name (StructError)
(node:4720) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Using named and default exports together. Consumers of your bundle will have to use jaysn['default'] to access
the default export, which may not be what you want. Use `exports: 'named'` to disable this warning
(node:4720) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: Error transforming bundle with 'uglify' plugin: Unexpected token: name (StructError)
Error: Error transforming bundle with 'uglify' plugin: Unexpected token: name (StructError)
Done in 13.40s.
```

This repo I was try to build https://github.com/lowsprofile/jaysn

This is my package.json
```json
{
  "name": "jaysn",
  "version": "0.1.3",
  "description": "Lightweight JSON database for Node, Hybrid, and browser.",
  "homepage": "https://lowsprofile.github.io/jaysn/",
  "repository": "lowsprofile/jaysn",
  "author": "Eries Trisnadi <me@eries.id> (https://eries.id)",
  "license": "MIT",
  "keywords": [],
  "main": "dist/jaysn.js",
  "module": "dist/jaysn.es.js",
  "eslintConfig": {
    "parser": "babel-eslint",
    "extends": "airbnb-base"
  },
  "files": [
    "dist",
    "README.md",
    "LICENSE"
  ],
  "source": [
    "src/Jaysn.js"
  ],
  "dependencies": {
    "immutable": "~4.0.0-rc.9",
    "path": "^0.12.7",
    "superstruct": "^0.4.5"
  },
  "devDependencies": {
    "babel-cli": "^6.26.0",
    "babel-eslint": "^8.0.2",
    "babel-plugin-transform-es2015-destructuring": "^6.23.0",
    "babel-plugin-transform-es2015-function-name": "^6.24.1",
    "babel-plugin-transform-es2015-parameters": "^6.24.1",
    "babel-preset-es2015": "6.24.1",
    "babel-preset-es2015-rollup": "^3.0.0",
    "babel-preset-stage-1": "^6.24.1",
    "babel-register": "^6.26.0",
    "chai": "^4.1.2",
    "coveralls": "^3.0.0",
    "eslint": "^4.12.0",
    "eslint-config-airbnb-base": "^12.1.0",
    "eslint-config-prettier": "^2.9.0",
    "eslint-plugin-import": "^2.8.0",
    "eslint-plugin-prettier": "^2.4.0",
    "microbundle": "^0.2.4",
    "mocha": "^4.0.1",
    "mustache": "^2.3.0",
    "npm-run-all": "^4.1.2",
    "nyc": "^11.3.0",
    "prettier": "1.9.2",
    "rollup": "^0.52.0",
    "rollup-plugin-babel": "^3.0.2",
    "rollup-plugin-buble": "^0.18.0",
    "rollup-plugin-commonjs": "^8.2.6",
    "rollup-plugin-json": "^2.3.0",
    "rollup-plugin-node-resolve": "^3.0.0",
    "rollup-plugin-strip-banner": "^0.2.0",
    "rollup-plugin-uglify": "^2.0.1",
    "shx": "^0.2.2",
    "sinon": "^4.1.2",
    "uglify-es": "^3.2.0",
    "uglify-save-license": "^0.4.1"
  },
  "scripts": {
    "build": "run-s format build:*",
    "build:dist": "run-s clean:dist bundle format:dist",
    "build:docs": "run-s clean:docs mustache",
    "bundle": "run-s bundle:*",
    "bundle:cjs": "run-s \"rollup -c tools/build-cjs.js\"",
    "bundle:es": "run-s \"rollup -c tools/build-es.js\"",
    "format": "run-s \"lint --fix\"",
    "format:dist": "prettier --write \"dist/**/{jaysn.es.js,jaysn.js}\"",
    "clean:dist": "run-s \"rm -rf dist\"",
    "clean:docs": "run-s \"rm -rf docs\" \"mkdir docs\"",
    "coverage": "nyc report --reporter=text-lcov | coveralls",
    "lint": "eslint src test tools",
    "microbundle": "microbundle build",
    "mkdir": "shx mkdir",
    "mustache": "node tools/build-docs && shx cp -r resources/css docs && mustache -p resources/partials/menu.mustache -p resources/partials/content.mustache resources/docs.json resources/docs.mustache > docs/index.html",
    "prepublish": "run-s build",
    "rm": "shx rm",
    "rollup": "rollup",
    "test": "run-s -l lint \"test:mocha {@}\" test:cover --",
    "test:mocha": "mocha --require babel-register",
    "test:cover": "nyc --require babel-register mocha --timeout=3000"
  }
}
```

**Comment by developit** [comment]
seems like uglify is trying to process ES exports, which is odd. what happens if you uninstall uglify-es?

**Comment by lowsprofile** [comment]
Hi @developit this is what happens when I uninstall `uglify-es` and it still same.
```sh
$ yarn remove uglify-es
yarn remove v1.1.0
[1/2] Removing module uglify-es...
[2/2] Regenerating lockfile and installing missing dependencies...
success Uninstalled packages.
Done in 12.24s.
$ yarn microbundle
yarn run v1.1.0
$ microbundle build
Using named and default exports together. Consumers of your bundle will have to use jaysn['default'] to access the default export, which may not be what you want. Use `exports: 'named'` to disable this warning
(node:2996) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Error transforming bundle with 'uglify' plugin: Unexpected token: name (StructError)
(node:2996) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Using named and default exports together. Consumers of your bundle will have to use jaysn['default'] to access the default export, which may not be what you want. Use `exports: 'named'` to disable this warning
(node:2996) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: Error transforming bundle with 'uglify' plugin: Unexpected token: name (StructError)
Error: Error transforming bundle with 'uglify' plugin: Unexpected token: name
(StructError)
Done in 14.47s.
```

**Comment by developit** [comment]
What OS btw?

**Comment by lowsprofile** [comment]
Windows 7, is that a prob? I will try on my Ubuntu then. Will be back, and report what it show on my Ubuntu.

**Comment by boneskull** [comment]
I'm having this issue on macOS.  [This is why](https://www.npmjs.com/package/rollup-plugin-uglify#warning)

**Comment by developit** [comment]
Fixed in [0.4.3](https://github.com/developit/microbundle/releases/tag/0.4.3), which always uses uglify-es.
